prow_config.yaml

```diff
@@ -4,3 +4,20 @@ workflows:
   - app_dir: kubeflow/testing/workflows
     component: workflows
     name: unittests
+
+  - app_dir: kubeflow/kubeflow/testing/workflows
+    component: click_deploy_test
+    name: skipmeplease
+    job_types:
+    - periodic
+    include_dirs:
+    - bootstrap/*
```

Comment:
Have a single include_dirs and set it to something like
- some/nonexistent/dir

This way its clear we don't expect any files to match so tit will always be skipped in presubmit.

Once you make the above changes you could do the same thing but with a py_func and point it to some nonexistent py_func.

This way we also verify skipping is applied to py_funcs.

Reply:
Done.